app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)

```

Comment:
According to the checklist, it is recommended to use `()` instead of `\` for line continuation. Although both are correct, using parentheses is more readable and Pythonic.